File: discover_ruby/write_them_to_disc.rb
#!/usr/bin/ruby

require "net/http"
require "uri"

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 6e73c90fdfe07bca58b4c3e20eaa15e367169857'
}

uri = URI.parse("https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc")

response = Net::HTTP.get(uri)

puts response

f = File.new("/tmp/49", "w+")
f.write(response)

File: discover_more_python/write_them_to_disc.py
import urllib2
import os.path

request_headers = {'User-Agent': 'Holberton_School', 'Authorization': 'token 95fd848656afb3a7d20af06365375c2a6f0b2fa8'}

githuburl = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'
req = urllib2.Request(githuburl, headers=request_headers)
response = urllib2.urlopen(req)

#print response.read()

if os.path.exists("/tmp/49"):
    f = open("/tmp/49", "w")
    f.truncate()
    f.close()

varfile = open("/tmp/49", "w")

varfile.write(response.read())
varfile.close()
print "The file was saved!"
